### DISCUSSION TITLE
v1.6.7

### DISCUSSION THREAD

**Discussion by smira** [siderolabs/talos#8475] [General]
## [Talos 1.6.7](https://github.com/siderolabs/talos/releases/tag/v1.6.7) (2024-03-20)

Welcome to the v1.6.7 release of Talos!

Please try out the release binaries and report any issues at
https://github.com/siderolabs/talos/issues.

### Component Updates

* Linux: 6.1.82
* Kubernetes: 1.29.3

Talos is built with Go 1.21.8.

### Contributors

* Andrey Smirnov
* Utku Ozdemir
* Noel Georgi

### Changes
<details><summary>7 commits</summary>
<p>

* siderolabs/talos@46c8ac102 release(v1.6.7): prepare release
* siderolabs/talos@9ef06f60f fix: service lifecycle issues
* siderolabs/talos@2c9159977 fix: patch correctly config in `talosctl upgrade-k8s`
* siderolabs/talos@16691dfd5 fix: remove maintenance config when maintenance service is shut down
* siderolabs/talos@5cbbbfa68 fix: fix nil panic on maintenance upgrade with partial config
* siderolabs/talos@3c942fe9d fix: etcd config validation for worker
* siderolabs/talos@a5920a157 feat: update Kubernetes to 1.29.3, Linux to 6.1.82
</p>
</details>

### Changes from siderolabs/pkgs
<details><summary>1 commit</summary>
<p>

* siderolabs/pkgs@df44f94 feat: update dependencies for Talos 1.6.7
</p>
</details>

### Dependency Changes

* **github.com/siderolabs/pkgs**                 v1.6.0-26-g2961472 -> v1.6.0-27-gdf44f94
* **github.com/siderolabs/talos/pkg/machinery**  v1.6.6 -> v1.6.7
* **google.golang.org/protobuf**                 v1.31.0 -> v1.33.0
* **k8s.io/api**                                 v0.29.2 -> v0.29.3
* **k8s.io/apimachinery**                        v0.29.2 -> v0.29.3
* **k8s.io/apiserver**                           v0.29.2 -> v0.29.3
* **k8s.io/client-go**                           v0.29.2 -> v0.29.3
* **k8s.io/component-base**                      v0.29.2 -> v0.29.3
* **k8s.io/cri-api**                             v0.29.2 -> v0.29.3
* **k8s.io/kube-scheduler**                      v0.29.2 -> v0.29.3
* **k8s.io/kubectl**                             v0.29.2 -> v0.29.3
* **k8s.io/kubelet**                             v0.29.2 -> v0.29.3
* **k8s.io/pod-security-admission**              v0.29.2 -> v0.29.3

Previous release can be found at [v1.6.6](https://github.com/siderolabs/talos/releases/tag/v1.6.6)

## Images

```
ghcr.io/siderolabs/flannel:v0.23.0
ghcr.io/siderolabs/install-cni:v1.6.0-2-g9234398
registry.k8s.io/coredns/coredns:v1.11.1
gcr.io/etcd-development/etcd:v3.5.11
registry.k8s.io/kube-apiserver:v1.29.3
registry.k8s.io/kube-controller-manager:v1.29.3
registry.k8s.io/kube-scheduler:v1.29.3
registry.k8s.io/kube-proxy:v1.29.3
ghcr.io/siderolabs/kubelet:v1.29.3
ghcr.io/siderolabs/installer:v1.6.7
registry.k8s.io/pause:3.8
```

<hr /><em>This discussion was created from the release <a href='https://github.com/siderolabs/talos/releases/tag/v1.6.7'>v1.6.7</a>.</em>

**Comment by lucasmaurice** [comment]
I think their is a typo in the release description: 

> v0.29.2 -> v0.29.3 

Instead of:

> v1.29.2 -> v1.29.3

**Reply by frezbo** [reply]
That's correct, k8s go api is versioned as `v0.29.3`
